README.md

```diff
@@ -61,88 +60,84 @@ Installing the NPM package locally creates a set of static Office.js files in th
 
 For example, if you add the contents of the `dist` folder to the `assets/office-js` directory of your project, then you'd add the following `<script>` tag to your HTML file:
 
-    <script src="/assets/office-js/office.js"></script>
-
+```html
+<script src="/assets/office-js/office.js"></script>
+```
 <br />
 
 ## IntelliSense definitions
 
 TypeScript definitions for Office.js are available.
 
-* For latest **RELEASE** version of Office.js:
-  * DefinitelyTyped: <https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/office-js/index.d.ts>
-  * @types: `@types/office-js`.  (Acquire as `npm install @types/office-js --save-dev`)
-
-* For **any version** (including **RELEASE**, but also including **BETA**, etc.):
-  * Inside of the NPM package, under `dist/office.d.ts`
-  * In this repo: [dist/office.d.ts](dist/office.d.ts)
+- For latest **RELEASE** version of Office.js:
+  - DefinitelyTyped: <https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/office-js/index.d.ts>
+  - @types: `@types/office-js`.  (Acquire as `npm install @types/office-js --save-dev`)
+- For latest **BETA** version of Office.js:
+  - DefinitelyTyped: <https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/office-js-preview/index.d.ts>
+  - @types: `@types/office-js-preview`.  (Acquire as `npm install @types/office-js-preview --save-dev`)
+- For **any** version:
+  - Inside of the NPM package, under `dist/office.d.ts`
+  - In this repo: [dist/office.d.ts](dist/office.d.ts)
 
 ### Using TypeScript definitions with the NPM package
 
-1. If you are using the Office.js NPM package for the [firewall scenario](#target-scenarios) and want a d.ts file that precisely matches the JS contents, use the d.ts file that is located within the `/dist/office.d.ts` folder of the NPM package. You can achieve this by using a [triple-slash reference](https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html). 
+If you are using the Office.js NPM package for the [firewall scenario](#target-scenarios) and want a d.ts file that precisely matches the JS contents, use the d.ts file that is located within the `/dist/office.d.ts` folder of the NPM package. You can achieve this by using a [triple-slash reference](https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html). 
```

Comment:
```suggestion
If you're using the Office.js NPM package for the [firewall scenario](#target-scenarios) and want a d.ts file that precisely matches the JS contents, use the d.ts file that is located within the `/dist/office.d.ts` folder of the NPM package. Achieve this by using a [triple-slash reference](https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html). 
```
This is per Microsoft Voice and Style guide:
1. Project friendliness
2. Revise weak wrting
https://styleguides.azurewebsites.net/Styleguide/Read?id=2696&topicid=29324
or https://docs.microsoft.com/en-us/style-guide/top-10-tips-style-voice